test-integration/src/test/java/org/frankframework/filesystem/MailSendingTestHelper.java

```diff
@@ -140,7 +140,7 @@ public void _commitFile() throws Exception {
 		email.addSubElement(message);
 		message.setValue(StreamUtil.streamToString(currentFile.getInputStream()));
 
-		Message msg=new Message(email.toXML());
+		Message msg=new Message(email.asXmlString());
```

Comment:
is gelukkig dode code maar deze had mooier gekund ;)

Reply:
Eens, aangepast naar asMessage